doc/api/readline.md

```diff
@@ -358,6 +358,9 @@ added: v0.1.98
     only if `terminal` is set to `true` by the user or by an internal `output`
     check, otherwise the history caching mechanism is not initialized at all.
   * `prompt` - the prompt string to use. Default: `'> '`
+  * `crlfDelay` {number} If the delay between `\r` and `\n` exceeds
+    `crlfDelay`, both `\r` and `\n` will be treated as separate end-of-line
```

Comment:
I am afraid to put 10ms (seems low because of context switches and other factors). Now I set `[100, 2000]` as range.  It will go few more rounds to come up with better range 😄